tasks.py

```diff
@@ -0,0 +1,426 @@
+from invoke import task, run 
+import platform 
+import os 
+
+PLATFORM=platform.system()
```

Comment:
Please run `black` on this file - it will autoformat for you so you don't have to worry about manually following PEP8

Reply:
what is **black**? First time to hear that!